diff --git a/dist/server/lib/router-server.js b/dist/server/lib/router-server.js
index 81142b250240ee62552eb1e079f95e5823637180..3d8b493443a015706edc457da6e1828cc93d9eea 100644
--- a/dist/server/lib/router-server.js
+++ b/dist/server/lib/router-server.js
@@ -479,9 +479,9 @@ async function initialize(opts) {
             });
             // TODO: allow upgrade requests to pages/app paths?
             // this was not previously supported
-            if (matchedOutput) {
-                return socket.end();
-            }
+            // if (matchedOutput) {
+            //     return socket.end();
+            // }
             if (parsedUrl.protocol) {
                 return await (0, _proxyrequest.proxyRequest)(req, socket, parsedUrl, head);
             }
