diff --git a/dist/server/lib/router-server.js b/dist/server/lib/router-server.js
index 53b7745..bfeda98 100644
--- a/dist/server/lib/router-server.js
+++ b/dist/server/lib/router-server.js
@@ -720,9 +720,9 @@ async function initialize(opts) {
             });
             // TODO: allow upgrade requests to pages/app paths?
             // this was not previously supported
-            if (matchedOutput) {
-                return socket.end();
-            }
+            // if (matchedOutput) {
+            //     return socket.end();
+            // }
             if (finished && parsedUrl.protocol) {
                 if (!statusCode) {
                     return await (0, _proxyrequest.proxyRequest)(req, socket, parsedUrl, head);
