diff --git a/dist/server/lib/router-server.js b/dist/server/lib/router-server.js
index a71e5a5da32c237e0b439386bc4c3fd86dc4e845..2df30904b72145ab1805082b76eeba9143cd7662 100644
--- a/dist/server/lib/router-server.js
+++ b/dist/server/lib/router-server.js
@@ -621,9 +621,9 @@ async function initialize(opts) {
             });
             // TODO: allow upgrade requests to pages/app paths?
             // this was not previously supported
-            if (matchedOutput) {
-                return socket.end();
-            }
+            // if (matchedOutput) {
+            //     return socket.end();
+            // }
             if (finished && parsedUrl.protocol) {
                 if (!statusCode) {
                     return await (0, _proxyrequest.proxyRequest)(req, socket, parsedUrl, head);
