diff --git a/dist/server/lib/router-server.js b/dist/server/lib/router-server.js
index 35ea234431bc5171632c986f75dfbe0491de9bc2..80d8d3e8f4b4311d9d4dbb7be0b74cef10419568 100644
--- a/dist/server/lib/router-server.js
+++ b/dist/server/lib/router-server.js
@@ -621,9 +621,9 @@ async function initialize(opts) {
             });
             // TODO: allow upgrade requests to pages/app paths?
             // this was not previously supported
-            if (matchedOutput) {
-                return socket.end();
-            }
+            // if (matchedOutput) {
+            //     return socket.end();
+            // }
             if (parsedUrl.protocol) {
                 return await (0, _proxyrequest.proxyRequest)(req, socket, parsedUrl, head);
             }
